Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,168 @@
+      |> \.backgroundColor .~ UIColor.blue
+
+    _ = self.spacerView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    _ = self.pledgeImageView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+      |> \.backgroundColor .~ UIColor.orange
+
+    _ = self.descriptionStackView
+      |> descriptionStackViewStyle
+
+    _ = self.estimatedDeliveryLabel
+      |> estimatedDeliveryLabelStyle
+
+    _ = self.dateLabel
+      |> dateLabelStyle
+
+    _ = self.descriptionLabel
+      |> descriptionLabelStyle
+
+    _ = self.learnMoreLabel
+      |> learnMoreLabelStyle
+  }
+
+  // MARK: - Configuration
+
+  func configureWith(value: String) {
+    _ = self.dateLabel
+      |> \.text .~ value
+  }
+
+  // MARK: - Styles
+
+  private func arrangeStackView() {
+    self.spacerView.heightAnchor.constraint(equalToConstant: 10.0).isActive = true
+
+    [
+      self.spacerView,
+      self.estimatedDeliveryLabel,
+      self.dateLabel,
+      self.descriptionLabel,
+      self.learnMoreLabel
+    ].forEach(self.descriptionStackView.addArrangedSubview)
+
+    if #available(iOS 11.0, *) {
+      self.descriptionStackView.setCustomSpacing(10.0, after: self.dateLabel)
+    } else {
+      let view = UIView(frame: .zero)
+      view.translatesAutoresizingMaskIntoConstraints = true
+      view.heightAnchor.constraint(equalToConstant: 10.0).isActive = true
+      self.descriptionStackView.insertArrangedSubview(view, at: 3)
+    }
+
+    self.rootStackView.addArrangedSubview(self.descriptionStackView)
+  }
+}
+
+private let rootStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> \.alignment .~ UIStackView.Alignment.top
+    |> \.axis .~ NSLayoutConstraint.Axis.horizontal
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+    |> \.layoutMargins .~ UIEdgeInsets.init(topBottom: Styles.grid(5), leftRight: Styles.grid(2))
+    |> \.spacing .~ Styles.grid(2)
+}
+
+private let descriptionStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> \.axis .~ NSLayoutConstraint.Axis.vertical
+    |> \.distribution .~ UIStackView.Distribution.fill
+}
+
+private let estimatedDeliveryLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.text %~ { _ in Strings.Estimated_delivery_of() }
+    |> \.textColor .~ UIColor.ksr_text_dark_grey_500
+    |> \.font .~ UIFont.ksr_headline()
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.numberOfLines .~ 0
+}
+
+private let dateLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.textColor .~ UIColor.ksr_soft_black
+    |> \.font .~ UIFont.ksr_headline()
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.numberOfLines .~ 0
+}
+
+private let descriptionLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.text  %~ { _ in Strings.Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life() }
```

Comment:
So before this string makes it back through Crowdin we should temporarily use this when setting it on the `DeprecatedPledgeVC`

```
localizedString(
  key: "Kickstarter_is_not_a_store", 
  defaultValue: "Kickstarter is not a store."
)
```

Unfortunately we will lose German, French and other translations but since it's only temporary it should be fine...I would also create a follow up card in Trello (i.e. `Localize Kickstarter_is_not_a_store`) so that we won't forget about this